operator/test/test.sh

```diff
@@ -149,17 +149,27 @@ test_pipelines() {
   # change ubuntu image to ubi to avoid dockerhub registry pull limit
   PIPELINE_RUN=$(curl --fail --silent "$BASE_URL/examples/v1beta1/$manifest" | sed 's|ubuntu|registry.access.redhat.com/ubi8/ubi-minimal:latest|' | sed '/serviceAccountName/d' | KUBECONFIG="$KUBECONFIG_KCP" kubectl create -f -)
   echo "$PIPELINE_RUN"
-  
-  KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+
+  # KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+  local wait_period=0
```

Comment:
I'm not sure I understand what this change is necessary. You seem to be reimplementing the `kubectl wait` behavior.

Wouldn't the following change be preferable?
```
if ! KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=360s >/dev/null; then
  echo "[ERROR] Failed to execute Pipelinerun" >&2
  KUBECONFIG="$KUBECONFIG_KCP" get pipelinerun -o json
  exit 1
fi
```

Reply:
Same as before, this is for debugging, it's removed.